### PR TITLE
New package: ipv6calc-2.0.0

### PR DESCRIPTION
**Pull request by lonwillett**

### CODE REVIEW

File: srcpkgs/ipv6calc/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'ipv6calc'
+pkgname=ipv6calc
+version=2.0.0
+revision=1
+build_style=gnu-configure
+configure_args="--enable-geoip"
+hostmakedepends="perl"
+makedepends="geoip-devel"
+depends="geoip"
```
Comment (maxice8) [inline srcpkgs/ipv6calc/template]: should be found via shlibs

Comment (maxice8) [reply srcpkgs/ipv6calc/template]: or rather libgeoip

Comment (lonwillett) [reply srcpkgs/ipv6calc/template]: Right you are. Changed.

Comment (maxice8) [reply srcpkgs/ipv6calc/template]: i'll assume you checked that it doesn't use any of the executables provided by `geoip` and `libgeoip` is enough.

Comment (lonwillett) [reply srcpkgs/ipv6calc/template]: Yes. I wasn't really thinking when I added geoip to the dependencies. It should have been libgeoip.

**Pull request merged**